src/lang/de.json

```diff
@@ -151,6 +151,8 @@
     "night": "Nacht",
     "warm": "Warm",
     "cold": "Kalt",
+    "vome": "vome",
+    "fass": "fass",
```

Comment:
could you use the title case versions of these so the transformation is more of a backup?